### ISSUE TITLE
Error on a fresh Angular install (14.1.3), can't lint

### ISSUE THREAD

**Issue by fabioemoutinho** [angular-eslint/angular-eslint#1111]
Reproduction Repository: https://github.com/bitovi/angular-ngrx-chat/tree/steps
On a fresh angular install, running ng lint to install angular-eslint and then running again to run the linting step, gives the error below:

```
(node:10974) UnhandledPromiseRejectionWarning: TypeError: iterator.next is not a function
    at recurse (/path/node_modules/@nrwl/devkit/src/utils/convert-nx-executor.js:54:26)
    at /path/node_modules/@nrwl/devkit/src/utils/convert-nx-executor.js:71:17**
(Use `node --trace-warnings ...` to show where the warning was created)
(node:10974) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:10974) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@angular-eslint/builder`          | `14.0.2` |
| `ESLint`                           | `8.18.0` |
| `node`                             | `14.17.6` |

```sh
# Please run `npx ng version` in your project and paste the full output here:
Angular CLI: 14.1.3
Node: 14.17.6
Package Manager: npm 6.14.15 
OS: darwin x64

Angular: 14.1.3
... animations, cli, common, compiler, compiler-cli, core, forms
... platform-browser, platform-browser-dynamic, router

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1401.3
@angular-devkit/build-angular   14.1.3
@angular-devkit/core            14.1.3
@angular-devkit/schematics      14.1.3
@schematics/angular             14.1.3
rxjs                            7.5.6
typescript                      4.7.4

```

- [x] I have updated to the latest supported version of the packages and checked my `ng version` output per the instructions given here.

**Comment by AliKholmatov** [comment]
any updates?

**Comment by JamesHenry** [comment]
The fix has been merged into Nx, just waiting on the patch to be released. I believe pinning to Angular 14.1.2 is the workaround until then

**Comment by AlexandrTichon** [comment]
Same eror

**Comment by JamesHenry** [comment]
I've published `14.0.3` of the `@angular-eslint` packages and there are no longer any issues with fresh installs on Angular 14.1.3

**Comment by elcreator** [comment]
(Use `node --trace-warnings ...` to show where the warning was created)
(node:8780) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:8780) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.    

Angular: 14.2.0
Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1402.1
@angular-devkit/build-angular   14.2.1
@angular-devkit/core            14.2.1
@angular-devkit/schematics      14.2.1
@angular/cli                    14.2.1
@schematics/angular             14.2.1
rxjs                            7.5.6
typescript                      4.7.4

**Comment by PavelSkopik** [comment]
I also confirm that the issue exists with Angular 14.2.0. @
